.github/workflows/main.yml

```diff
@@ -79,6 +80,33 @@ jobs:
           path: ./*
           key: ${{ github.ref }}-${{ github.sha }}-build-release
 
+  build-docs:
```

Comment:
This job is by far the longest running one on the CI, let's see if we can speed it up.

Reply:
Requested changes done 🙏